### PR TITLE
Update to discord.js v13 (third try)

### PR DESCRIPTION
**Pull request by violine1101**
Previous PRs:
#224, #236

Reverted in:
#235, #243

This change requires more testing and changes to the codebase to ensure it works properly with discord.js v13.

### CODE REVIEW

**Comment by mdashlw** [comment]
What's the status of this?

**Comment by violine1101** [comment]
Essentially, upgrading to discordjs v13 broke stuff and we don't know why it broke stuff or how to fix it.

I'm currently in the process of rebasing everything, and then I'll merge this after having tested it locally.

**Comment by chandler05** [comment]
I noticed that reactions to the pending request seem to appear slower when a request is added. Is this just my bot or does it happen for yours as well?

**Review by chandler05 — APPROVED** [review]

**Comment by violine1101** [comment]
I'm not sure. It's always a little delayed, however possibly the delay has been increased with v13. I think for our purposes it's fine.

In the future it'd be nice to replace this with buttons anyway, but we aren't that far yet.

File: src/MojiraBot.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export default class MojiraBot {
 	public static jira = new JiraClient( {
 		host: 'https://bugs.mojang.com',
 		telemetry: false,
+		newErrorHandling: true
```
Comment (violine1101) [inline src/MojiraBot.ts]: I looked into changing this but feared that it might mess up error reporting.

If possible we should probably do this in a separate PR.

**Comment by violine1101** [comment]
Forgot to include co-authors in the commit, new commit is 20ee09e08777f826b3c708a69bcff814e8911e34 (no other changes)
